.github/workflows/_build_artifacts.yml

```diff
@@ -60,6 +59,7 @@ jobs:
         with:
           version: ${{ env.VERSION }}
       - name: Delete stale artifacts
+        if: inputs.profile == 'release' && inputs.stage == 'release'
```

Comment:
```suggestion
        # Build artifacts are only uploaded to the drafted release on `main` release builds, so only clear them in those cases too to prevent PRs from removing drafted release assets.
        if: inputs.profile == 'release' && inputs.stage == 'release'
```